mne/io/brainvision/tests/test_brainvision.py

```diff
@@ -77,82 +77,50 @@ def test_orig_units(recwarn):
-    with pytest.warns(RuntimeWarning, match='coordinate information'):
-        raw = read_raw_brainvision(vhdr_v2_path)
-    assert raw.info['meas_date'] is None
-    assert 'unspecified' in repr(raw.info)
+    # Return header and marker file paths
+    tmp_vhdr_file = op.join(tmpdir, op.basename(vhdr_path))
+    tmp_vmrk_file = op.join(tmpdir, op.basename(vmrk_path))
 
-    # Test files with faulty dates introduced by segmenting a file without
-    # date information. Should not raise a strptime ValueError
-    raw = read_raw_brainvision(vhdr_bad_date)
-    assert raw.info['meas_date'] is None
-    assert 'unspecified' in repr(raw.info)
-
-    # Reuse our vmrk to test several other cases with dates
-    tmpdir = _TempDir()
-    tmp_vhdr_file = op.join(tmpdir, op.basename(vhdr_bad_date))
-    tmp_vmrk_file = tmp_vhdr_file.replace('.vhdr', '.vmrk')
-    tmp_eeg_file = op.join(tmpdir, op.basename(eeg_path))
-    sh.copyfile(vhdr_bad_date, tmp_vhdr_file)
-    sh.copyfile(vhdr_bad_date.replace('.vhdr', '.vmrk'), tmp_vmrk_file)
-    sh.copyfile(eeg_path, tmp_eeg_file)
-
-    # We'll exclusively manipulate the "New Segment" line in vmrk
-    with open(tmp_vmrk_file, 'r') as fin:
-        lines = fin.readlines()
-    idx = lines.index('Mk1=New Segment,,1,1,0,00000000000304125000\n')
+    return tmp_vhdr_file, tmp_vmrk_file, lines, idx
 
-    # Now perform some tests
-    # Test that we get no error for trying to extract meas_date if there is
-    # no marker of type "New Segment" in the data
-    lines[idx] = 'Mk1=STATUS,,1,1,0\n'
-    with open(tmp_vmrk_file, 'w') as fout:
-        fout.writelines(lines)
-    raw = read_raw_brainvision(tmp_vhdr_file)
-    assert raw.info['meas_date'] is None
-    assert 'unspecified' in repr(raw.info)
 
-    # Test that we extract no date, if "New Segment", but no date specified
-    # Note the trailing comma but missing data following that comma
-    lines[idx] = 'Mk1=New Segment,,1,1,0,\n'
-    with open(tmp_vmrk_file, 'w') as fout:
-        fout.writelines(lines)
-    raw = read_raw_brainvision(tmp_vhdr_file)
-    assert raw.info['meas_date'] is None
-    assert 'unspecified' in repr(raw.info)
+@pytest.mark.parametrize('newstring, expected', [
+    pytest.param('Mk1=New Segment,,1,1,0,20131113161403794232\n', '2013-11-13 16:14:03 GMT'),  # noqa: E501
+    pytest.param('Mk1=New Segment,,1,1,0,20070716122240937454\nMk2=New Segment,,2,1,0,20070716122240937455\n', '2007-07-16 12:22:40 GMT'),  # noqa: E501
+    pytest.param('Mk1=STATUS,,1,1,0\n', 'unspecified'),
+    pytest.param('Mk1=New Segment,,1,1,0,\n', 'unspecified'),
+    pytest.param('Mk1=New Segment,,1,1,0\n', 'unspecified'),
+    pytest.param('Mk1=New Segment,,1,1,0,00000000000304125000', 'unspecified'),
+    pytest.param('Mk1=New Segment,,1,1,0,\nMk2=New Segment,,2,1,0,20070716122240937454\n', '2007-07-16 12:22:40 GMT'),  # noqa: E501
+])
+def test_dates(_get_date_test_data, newstring, expected):
```

Comment:
`new_string` might not be informative enough. I would call it `mocking_date` and I would also change `expected` for `EXPECTED_DATE_REPRESENTATION` or `EXPECTED_DATE__REPR__`